Building release v2.2.1 from tarball always fails

TL;DR: Building ngx_mruby v2.2.1 from release tarball fails because `build_mruby` target in `Makefile.in` tries to look at `.git` and `.mruby-version` but both of them are not included in the tarball.

https://github.com/matsumotory/ngx_mruby/blob/7a3198c992caccde70af24485d487e39d6314423/Makefile.in#L75-L86

This is minimal Dockerfile to reproduce this problem:

```Dockerfile
FROM alpine

RUN apk add make ruby ruby-rake bison openssl gcc build-base curl tar nginx git

RUN curl -L http://nginx.org/download/nginx-1.18.0.tar.gz -o nginx.tar.gz
RUN tar xzvf nginx.tar.gz

RUN curl -L https://github.com/matsumotory/ngx_mruby/archive/v2.2.1.tar.gz -o ngx_mruby.tar.gz
RUN tar xzvf ngx_mruby.tar.gz

WORKDIR /ngx_mruby-2.2.1/

RUN ./configure --with-ngx-src-root=/nginx/
RUN make
```

This is the output I can get when building this Dockerfile:

```bash
% docker build .
(snip)
Step 9/9 : RUN make
 ---> Running in af5c6975ab1b
fatal: not a git repository (or any of the parent directories): .git
make: *** [Makefile:76: build_mruby] Error 128
```

I think this problem is related to #453.

--- a/Makefile.in
+++ b/Makefile.in
@@ -73,6 +73,7 @@ clobber: clean_mruby clean
 # mruby
 #
 build_mruby: $(MRUBY_BUILD_DEPEND_TARGETS)
+	# Only for ngx_mruby developers getting source code from git.
 	@if [ -f .mruby_version ]; then \
 		built_version=`cat .mruby_version`; \
 		current_version=`git log -1 --format="%H" mruby`; \
@@ -80,7 +81,7 @@ build_mruby: $(MRUBY_BUILD_DEPEND_TARGETS)
 			echo $$current_version > .mruby_version; \
 			(cd $(MRUBY_ROOT) && rm -rf build); \
 		fi; \
-	else \
+	elif [ -d .git ]; then \
 		git log -1 --format="%H" mruby > .mruby_version; \
 	fi
 	(cd $(MRUBY_ROOT) && $(RAKE) MRUBY_CONFIG=$(NGX_MRUBY_ROOT)/build_config.rb NGX_MRUBY_CFLAGS="$(NGX_MRUBY_CFLAGS)" NGX_MRUBY_LDFLAGS="$(NGX_MRUBY_LDFLAGS)")

